flutter/lib/src/web/web_sentry_js_binding.dart

```diff
@@ -22,6 +22,22 @@ class WebSentryJsBinding implements SentryJsBinding {
     _client = SentryJsClient();
   }
 
+  @override
+  void updateSession({int? errors, String? status}) {
+    final isolationScope = SentryJsIsolationScope();
+    JSObject? currentSession = isolationScope.getSession();
+
+    if (status != null) {
+      currentSession?['status'] = status.toJS;
+    }
+
+    if (errors != null) {
+      currentSession?['errors'] = errors.toJS;
+    }
+
+    isolationScope.setSession(currentSession!);
```

Comment:
We never checked if `currentSession` is non-null, but are force unwrapping it here.